plugins/backend/catalog-backend-module-aws/src/utils/tags.ts

```diff
@@ -46,7 +46,9 @@ export const labelsFromTags = (tags?: Tag[] | Record<string, string>) => {
       .reduce((acc: Record<string, string>, tag) => {
         if (tag.Key && tag.Value) {
           const key = tag.Key.replaceAll(':', '_').replaceAll('/', '-');
-          acc[key] = tag.Value.replaceAll('/', '-').substring(0, 63);
+          acc[key] = tag.Value.replaceAll('/', '-')
```

Comment:
I really wonder if we should be adding these as labels at all? These changes materially affect the value and they might not be meaningful / usable without them. We could perhaps use tags? e.g. tags: ['keyA=valueA'] ?

Reply:
Im not sure tbh.